docs/concepts/nodes.md

```diff
@@ -25,42 +25,24 @@ You're likely to see the term _node_ throughout the IPFS docs, issues, and relat
 
 There are different types of IPFS nodes. And depending on the use-case, a single IPFS node can serve one of many functions:
 
-- [Preload](#preload)
 - [Relay](#relay)
 - [Bootstrap](#bootstrap)
 - [Delegate routing](#delegate-routing-node)
 
-### Preload
-
-Use to make a UnixFS DAG publicly available by calling `ipfs refs -r <CID>` on a randomly chosen preload node's HTTP API. This puts the CID in the preload nodes' wantlist, causing it to fetch the data from the user. Other nodes requesting the content can then resolve it from the preload node using bitswap, as the data is now present in the preload node’s blockstore.
-
-#### Features of a preload node
-
-- They are Kubo nodes with API ports exposed. Some HTTP API commands are accessible.
-- Used by JS-IPFS nodes running in browser contexts.
-- JS-ipfs nodes remain connected to the libp2p swarm ports of all preload nodes by having preload nodes on the bootstrap list.
-- Often on the same _server_ as a [delegate routing node](#delegate-routing-node), though both the delegate routing service and preload service are addressed differently. This is done by having different multiaddrs that resolve to the same machine.
-- Preload nodes are in the default JS-IPFS configuration as bootstrap nodes, so they will maintain libp2p swarm connections to them at all times.
-    - They are configured as regular bootstrap nodes, but as a convention have the string 'preload' in their `/dnsaddr` multiaddrs.
-
-#### Limitations of a preload node:
-
-- Default preload nodes provided by Protocol Labs garbage collect every hour, so preloaded content only survives for that long. However, this is configurable. You can run nodes with customized policies.
-- Requires client to be smart about what gets preloaded: recursive preload of a big DAG.
-- Only works with dag-pb CIDs because that's all the refs command understands. It's harder to find non-dag-pb content, e.g., you need a connection to the publishing js-ipfs instance or it needs to be put on the DHT by a delegate node.
-
 ### Relay
 
 If an IPFS node deems itself unreachable by the public internet, IPFS nodes may choose to use a relay node as a kind of VPN in an attempt to reach the unreachable node.
 
 #### Features of a relay node
 
 - Implements either [v1](https://github.com/libp2p/specs/blob/master/relay/circuit-v1.md) or [v2](https://github.com/libp2p/specs/blob/master/relay/circuit-v2.md) of the Circuit Relay protocol.
-- Can be either Kubo or JS-IPFS nodes; however there are standalone implementations as well:
-  - [js-libp2p-relay-server](https://github.com/libp2p/js-libp2p-relay-server) (supports circuit v1)
+- Can be either Kubo or Helia nodes; however there are standalone implementations as well:
+  - [js-libp2p/circuit-relay](https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#setup-with-relay) (supports circuit v1 & v2) TODO_JS_IPFS_DEPRECATION - CONFIRM that js-libp2p still supports relay v1?
```

Comment:
@achingbrain does js-libp2p still support relay v1? I couldn't tell from looking at https://github.com/libp2p/js-libp2p/blob/master/packages/libp2p/src/circuit-relay/server/index.ts#L14

Reply:
nvm, we may remove references to relay v1..